projects/kit/components/stepper/stepper.component.ts

```diff
@@ -48,6 +50,8 @@ export class TuiStepperComponent {
     @ContentChildren(forwardRef(() => TuiStepComponent), {read: ElementRef})
     private readonly steps: QueryList<ElementRef<HTMLElement>> = EMPTY_QUERY;
 
+    private readonly changeDetector: ChangeDetectorRef = inject(ChangeDetectorRef);
```

Comment:
please, use old way for backward compatibility

```ts
constructor(
 private readonly cd: ChangeDetectorRef
) {}
```

Reply:
fixed